src/components/MazeMap/MazeMap.tsx

```diff
@@ -128,20 +131,62 @@ const MazeMap = (props: MazeMapProps) => {
     }
   };
 
+  const deleteIcons = (style: any) => {
+    // layer 143 Room / Point-Of-Interest icons
+    // layer 144 Building Icons
+    const poiIcons = style.layers[143];
+    const buildingIcons = style.layers[144];
+
+    // deletes all associated icon fields from style layer layout.
+    delete poiIcons.layout["icon-image"];
+    delete poiIcons.layout["icon-offset"];
+    delete poiIcons.layout["text-offset"];
+    delete poiIcons.layout["text-anchor"];
+
+    delete buildingIcons.layout["icon-image"];
+    delete buildingIcons.layout["icon-offset"];
+    delete buildingIcons.layout["text-offset"];
+    delete buildingIcons.layout["text-anchor"];
+
+    return style;
+  };
+
+  const deleteAll = (style: any) => {
```

Comment:
all doesn't seem to make up everything - does this include the watermark for example? this is a bit vague without any comments imo

Reply:
is that better?